project/Dependencies.scala

```diff
@@ -48,14 +48,17 @@ object Dependencies {
     val akkaHttpMetrics  = "1.7.1"
     val badRows          = "2.1.1"
     val log4cats         = "2.6.0"
+    val http4s           = "0.23.23"
+    val blaze            = "0.23.15"
+    val http4sNetty      = "0.5.9"
+    val decline          = "2.4.1"
+    val circe            = "0.13.0"
```

Comment:
Why not circe 0.14.1 and circeConfig 0.10.0?

Reply:
```
[info]   +-com.snowplowanalytics:snowplow-badrows_2.12:2.1.1
[info]   | +-com.snowplowanalytics:iglu-scala-client_2.12:1.1.1
[info]   | +-com.snowplowanalytics:snowplow-scala-analytics-sdk_2.12:2.1.0 [S]
[info]   | | +-com.snowplowanalytics:iglu-core-circe_2.12:1.0.0 [S]
[info]   | | | +-com.snowplowanalytics:iglu-core_2.12:1.0.0 [S]
[info]   | | | +-io.circe:circe-core_2.12:0.13.0 (evicted by: 0.14.1)
```
I just need to bump bad rows to 2.2.1 :+1: